src/jukebox/components/synchronisation/sync_shared/__init__.py

```diff
@@ -0,0 +1,355 @@
+"""
+Handles the synchronisation of shared audiofolder and the card database
```

Comment:
I am seeing a few redundant things, e.g. for me there shouldn't be a difference between the function `sync_card` and `sync_folder` 🤷‍♀️

I am curious.. is this a) a 2-way sync, b) Phoniebox -> Server or c) Server -> Phoniebox?

Reply:
Currently its only c) Server -> Phoniebox.

They are not quite the same. The audiofolders are just placed in the file system and mpd can performe the update after.
The card database behaves a bit different as its a singel file for all cards, and therefore a special treatment is needed to update correctly (and only if changes are present). See also the comment in the header, where i tried to describe this specialities.